fetch-core/dune

```diff
@@ -1,6 +1,5 @@
 (library
  (name fetch_core)
  (public_name fetch-core)
- (libraries cohttp)
```

Comment:
why this commit ? https://github.com/lessp/reason-fetch/pull/3/commits/ba623c0a0733a40fe083f1fd3a3229ad4e1f7f5d @lessp

Reply:
hmm, is the link to the right commit? @Et7f3

Reply:
for me we have two commit the same https://github.com/lessp/reason-fetch/pull/3/commits/937d1378a18c284436d467d7a1fb07586d23847b

Reply:
Ah it is a merge. A git pull -f sould be enough to avoid this commit 😂